src/main/java/org/datadog/jenkins/plugins/datadog/audit/DatadogAudit.java

```diff
@@ -0,0 +1,45 @@
+package org.datadog.jenkins.plugins.datadog.audit;
+
+
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+/**
+ * Class used to centralize some util methods to show telemetry metrics via Jenkins logs.
+ * This logger can be opt-in by the user in the Jenkins logs manager.
+ */
+public class DatadogAudit {
+
+    private static transient final Logger LOGGER = Logger.getLogger(DatadogAudit.class.getName());
+
+    static {
+        logFine("## DatadogAudit enabled ##");
+    }
+
+    private static void logFine(String msg) {
+        if(!LOGGER.isLoggable(Level.FINE)){
+            return;
+        }
+
+        LOGGER.fine(msg);
+    }
+
+    public static long currentTimeMillis() {
+        if(!LOGGER.isLoggable(Level.FINE)){
+            return -1L;
+        }
+
+        return System.currentTimeMillis();
+    }
+
+    public static void log(String msg, long start, long end) {
+        if(start == -1L || end == -1L){
+            return;
+        }
```

Comment:
I think the interface of this class can be improved. It's a bit weird having to rely on a wrapped version of `curentTimeMillis` that returns a sentinel value. What about something like this?

```java
public class DatadogAudit {
  public static void log(String msg);
  public static DatadogAuditStopwatch startWatch();
}

public class DatadogAuditStopwatch {
  public void stop(String msg) {
        long duration = System.currentTimeMillis() - this.start;
        if(duration > 10){
            DatadogAudit.logFine(msg +" [duration: "+duration+" ms, start: " + start + ", end: " + end+"]");
        }
  }
}
```

Or something like this:

```java
    public static void log(String msg, long start, long end) {
        if(!LOGGER.isLoggable(Level.FINE)){
            return;
        }
[...]
```